webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/index.js

```diff
@@ -94,17 +94,22 @@ const RegistrationCommandsPage = () => {
   // Plugins
   const [pluginValues, setPluginValues] = useState({});
 
-  const handlePluginValue = data => {
-    setPluginValues({ ...pluginValues, ...data });
-  };
+  const handlePluginValue = useCallback(data => {
+    setPluginValues(prevValues => ({ ...prevValues, ...data }));
+  }, []);
 
-  const handleInvalidField = (field, isValid) => {
+  const handleInvalidField = useCallback((field, isValid) => {
     if (isValid) {
-      setInvalidFields(invalidFields.filter(f => f !== field));
-    } else if (!invalidFields.find(f => f === field)) {
-      setInvalidFields([...invalidFields, field].sort());
+      setInvalidFields(prevFields => prevFields.filter(f => f !== field));
+    } else {
+      setInvalidFields(prevFields => {
+        if (!prevFields.find(f => f === field)) {
+          return [...prevFields, field].sort();
```

Comment:
I'm not sure why we need the fields to be sorted, but that's outside the scope of this PR